import sys

def error(message):
    print("Error: {}".format(message))
    sys.exit(1)
